.vscode/settings.json

```diff
@@ -7,4 +7,23 @@
     "formatting.local": "github.com/snyk/snyk-ls"
   },
   "go.toolsManagement.autoUpdate": true,
+  "workbench.colorCustomizations": {
```

Comment:
We all have different tastes for colour, I wouldn't enforce this on the repository level 😉

Reply:
Added accidentally, fixed in f8652008d577e6580ea4351b03770cec65c91cec